src/actions/quiz.client.ts

```diff
@@ -62,7 +62,7 @@ export async function generateQuiz({
     topicId,
   });
 
-  if (questions.length === 0) return { data: null };
+  if (questions.length === 0) return null;
```

Comment:
why is this change done?
this might cause errors
for example in page.tsx of new-quiz the response has been used as an object on line 33

Reply:
Oh, I did this because of new-quiz route error handling, will change back it